### PR TITLE
323 offender disability

### PR DESCRIPTION
**Pull request by andymarke**

### CODE REVIEW

File: src/main/java/uk/gov/justice/digital/delius/data/api/Disability.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.time.LocalDate;
+
+@Data
+@Builder
+public class Disability {
+    private Long disabilityId;
```
Comment (llmikeyj) [inline src/main/java/uk/gov/justice/digital/delius/data/api/Disability.java]: Why do we need to surface the Id through the API? Is this the PK or something else?

File: src/main/java/uk/gov/justice/digital/delius/data/api/Disability.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    private KeyValue type;
```
Comment (llmikeyj) [inline src/main/java/uk/gov/justice/digital/delius/data/api/Disability.java]: What does 'type' mean? Is there a better description?

File: src/test/java/uk/gov/justice/digital/delius/transformers/OffenderTransformerTest.java [hunk 3]
```diff
@@ -33,6 +40,90 @@ public void offenderManagerAllocationReasonNullWhenOffenderTransferAbsent() {
 
     }
 
+    @Test
+    public void disabilitiesCopiedWithLatestFirst() {
+        assertThat(idsOf(offenderTransformer.fullOffenderOf(anOffender()
```
Comment (llmikeyj) [inline src/test/java/uk/gov/justice/digital/delius/transformers/OffenderTransformerTest.java]: You don't really need idsOf(...), you can use something like assertThat(offenderTransformer.fullOffenderOf(..)).extracting("disabilityId").containsExactly(...)

File: src/main/java/uk/gov/justice/digital/delius/data/api/Disability.java (hunk 1 above)
Comment (andymarke) [reply src/main/java/uk/gov/justice/digital/delius/data/api/Disability.java]: Yeah, primary key for a disability - generally clients need something that can be used to distinguish between the disabilities - though are unlikely to use in subsequent API calls until we have update/delete functionality

File: src/main/java/uk/gov/justice/digital/delius/data/api/Disability.java (hunk 2 above)
Comment (andymarke) [reply src/main/java/uk/gov/justice/digital/delius/data/api/Disability.java]: Renamed to disabilityType

**Comment by llmikeyj** [comment]
lgtm!

**Pull request merged**